health/micro-ui-react19/web/packages/css/src/pages/employee/commodityManagement.scss

```
@@ -0,0 +1,492 @@
+
+.cm-shipment-field {
+  display: flex;
+  flex-direction: column;
+  gap: 0.25rem;
+}
+
+.cm-shipment-label {
+  color: #0B0C0C;
+  font-family: "Roboto";
+  font-weight: 400;
+  font-style: Regular;
```

Stylelint (17.6.0)

[error] Expected "Regular" to be "regular" (value-keyword-case)

(value-keyword-case)